plugin/trino-redshift/src/main/java/io/trino/plugin/redshift/RedshiftClient.java

```diff
@@ -667,10 +667,7 @@ public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connect
                         RedshiftClient::writeChar));
 
             case Types.VARCHAR: {
-                if (type.columnSize().isEmpty()) {
-                    throw new TrinoException(REDSHIFT_INVALID_TYPE, "column size not present");
-                }
-                int length = type.requiredColumnSize();
+                int length = type.columnSize().orElse(REDSHIFT_MAX_VARCHAR);
```

Comment:
Could you leave a code comment why we use `REDSHIFT_MAX_VARCHAR` when the column size doesn't exist?